Lib/idlelib/debugger.py

```diff
@@ -1,6 +1,6 @@
 """Debug user code with a GUI interface to a subclass of bdb.Bdb.
 
-The Idb idb and Debugger gui instances each need a reference to each
+The Idb and Debugger gui instances each need a reference to each
 other or to an rpc proxy for each other.
```

Comment:
'idb' and 'gui' are names of instances of class Idb and Debugger.  Lets make this clearer.
```suggestion
The Idb instance 'idb' and Debugger instance 'gui' need references to each
other or to an rpc proxy for each other.
```